specs/interop/eth-lockbox.md

```diff
@@ -0,0 +1,255 @@
+that a chain ends up with more liquidity than its `OptimismPortal`.
+The `ETHLockbox` improves the Superchain's interoperable ETH withdrawal user experience and avoids this issue.
+To do so, it unifies ETH L1 liquidity in a single contract (`ETHLockbox`), enabling seamless withdrawals of ETH
+from any OP chain in the Superchain, regardless of where the ETH was initially deposited.
+
+## Design
+
+The `ETHLockbox` contract is designed to manage the unified ETH liquidity for the Superchain.
+It implements two main functions: `lockETH` for depositing ETH into the lockbox,
+and `unlockETH` for withdrawing ETH from the lockbox.
+
+These functions are called by the `OptimismPortal` contracts to manage the shared ETH liquidity
+when making deposits or finalizing withdrawals.
+
+Authorization of `OptimismPortal`s is managed by the `ProxyAdmin` owner.
+The `ETHLockbox` contract is proxied and managed by the L1 `ProxyAdmin`.
+
+### Interface and properties
+
+#### `lockETH`
+
+Deposits and locks ETH into the lockbox's liquidity pool.
+
+- The function MUST accept ETH.
+- Only authorized `OptimismPortal` addresses MUST be allowed to interact.
+- The function MUST NOT revert when called by an authorized `OptimismPortal`
+- The function MUST emit the `ETHLocked` event with the `portal` that called it and the `amount`.
+
+```solidity
+function lockETH() external payable;
+```
+
+#### `unlockETH`
+
+Withdraws a specified amount of ETH from the lockbox's liquidity pool to the `OptimismPortal` calling it.
+
+- Only authorized `OptimismPortal` addresses MUST be allowed to interact.
+- The function MUST NOT revert when called by an authorized `OptimismPortal` unless paused.
+- The function MUST emit the `ETHUnlocked` event with the `portal` that called it and the `amount`.
+- The function MUST use `donateETH` when sending ETH to avoid triggering deposits.
+- The function MUST NOT allow to be called as part of a withdrawal transaction (`OptimismPortal.l2Sender()` MUST be the `DEFAULT_L2_SENDER`).
+
+```solidity
+function unlockETH(uint256 _value) external;
+```
+
+#### `authorizePortal`
+
+Authorizes an `OptimismPortal` to interact with the `ETHLockbox`.
+
+- Only the `ProxyAdmin` owner can call the function.
+- The function MUST emit the `PortalAuthorized` event with the `portal`.
+- The function MUST NOT allow the same `OptimismPortal` to be authorized more than once.
+
+```solidity
+function authorizePortal(address _portal) external;
+```
+
+#### `authorizeLockbox`
+
+Authorizes another `ETHLockbox` to migrate its ETH liquidity to the current `ETHLockbox`.
+
+- Only the `ProxyAdmin` owner can call the function.
+- The function MUST emit the `LockboxAuthorized` event with the `lockbox` that is being authorized.
+- The function MUST NOT allow the same `ETHLockbox` to be authorized more than once.
+
+```solidity
+function authorizeLockbox(address _lockbox) external;
+```
```

Comment:
Should these also have a variant of the `The `ProxyAdmin` owner of the source lockbox must be the same as the `ProxyAdmin` owner of the destination lockbox` requirement that `migrateLiquidity` has? For `authorizePortal` the portal and lockbox would have the same proxy admin owner, and for `authorizeLockbox` the two lockboxes would have the same proxy admin owner